setup.py

```diff
@@ -112,7 +150,39 @@ def runtest(self):
         cellprofiler.__main__.stop_cellprofiler()
 
         sys.exit(errno)
-
+        
+class CPPy2Exe(py2exe.build_exe.py2exe):
```

Comment:
Can this be achieved with the [data_files](http://www.py2exe.org/index.cgi/data_files) option?